FormidableLabs/babel-plugin-transform-define
# Update lodash to fix low vulnerability

> ### Description

Packages that use babel-plugin-transform-define are receiving a warning on npm audit of a Prototype Pollution problem. This is already fixed on lodash, so this commit updates the patch of it to version 4.17.10, so this warning will disappear.

> ### How to test

1. Run tests and check that nothing is broken
2. Create a draft project with `babel-plugin-transform-define` and run `npm audit` with and without this change. Check that there is a low vulnerability before, that is fixed with this PR.
